dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -476,6 +476,16 @@ export const AdSlot = ({
 				/>
 			);
 		}
+		case 'exclusion': {
```

Comment:
Minor - is there a way to make it clear this is for consentless only. Either naming or a comment?

Reply:
Good shout, I will add!